### PR TITLE
Update _index.html

### PR DESCRIPTION
**Pull request by rodrigoporcionato**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@rodrigoporcionato  You can make your emails match up in one of two ways:

1. Login to the [CLA site](https://identity.linuxfoundation.org/projects/cncf) and change it to match what you use as your default email address on your GitHub account
2. Change your default email address on your GH account to the same one you used to sign the CLA.

**Comment by kbhawkey** [comment]
@rodrigoporcionato ,
To move these changes forward, you need to rebase your changes.
Thanks!

File: content/pt/_index.html [hunk 1]
```diff
@@ -43,12 +43,12 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona on Maio 20-23, 2019</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona em Maio 20-23, 2019</a>
```
Comment (bl-ue) [inline content/pt/_index.html:46]: Based on the change below (line 51):
```suggestion
        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona em Maio 20-23 de 2019</a>
```

**Comment by femrtnz** [comment]
@rodrigoporcionato ca you check the conflicts and rebase?

thanks!

**Comment by rodrigoporcionato** [comment]
done

File: content/pt/_index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">KubeCon em Shanghai em June 24-26, 2019</a>
+        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">KubeCon em Shanghai em Junho, 24-26 de 2019</a>
```
Comment (kbhawkey) [inline content/pt/_index.html:50]: Hi @rodrigoporcionato .
Would you check this link again (See the `en` version)?
I think these links are updated as the new KubeCon dates are announced.

**Comment by irvifa** [comment]
Hi @rodrigoporcionato are you still working on this?

**Comment by yagonobre** [comment]
@rodrigoporcionato  can we close this pr or are you still working on it?

**Review by jcjesus — APPROVED** [review]

**Pull request merged**